CHANGELOG.md

```diff
@@ -14,9 +14,12 @@ Check [Keep a Changelog](http://keepachangelog.com/) for recommendations on how
 
 :microscope: - experimental
 
+## [unreleased]
+- :pencil: added page object end-to-end tests
```

Comment:
add it to 2.0.0. it will a part of release

Reply:
done